api/db-cdr/changelog/db.changelog-10.xml

```diff
@@ -0,0 +1,150 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
```

Comment:
Can you add a comment indicating what version of the CDM this is for?

Please verify on the resulting MySQL instance that the DDL looks the same as the OMOP DDL should.

Reply:
Good idea. Got it. Added comment with Omop cdm v5.2. Verified from the specs the columns are same.